massa-node/base_config/gas_costs/wasm_gas_costs.json

```diff
@@ -2,12 +2,12 @@
   "Wasm:Drop": 8,
   "Wasm:GlobalGet": 28,
   "Wasm:GlobalSet": 26,
-  "Wasm:I32Add": 0,
```

Comment:
This is not a problem if they are 0 as the amount of gas for a wasm operator is the mean of the values in this file